mapping-for-navigation/_posts/0000-01-09-modeling-interesections-for-map-navigation.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Modeling Intersections for Map Navigation
+---
+
+Capturing the various cases of Intersection modelling in OSM and the best practices to follow. The aim of this document is not go into the technicalities of traffic flow in a junction but rather to give a mapper clear cut instructions on how to model different intersections in OSM. Aiming to keep this document simple and easy to follow.
```

Comment:
While it’s important that we lay out various situations and how to deal with them, I think it also helps to start out with an overarching theme. In my opinion, a good rule of thumb is: keep the intersection as simple as possible while accurately reflecting physical separations.